### DISCUSSION TITLE
make image/audio/video support DataURI

### DISCUSSION THREAD

**Discussion by liunux4odoo** [zauberzeug/nicegui#1157] [Ideas / Feature Requests]
## Why?
The image/audio/video consume urls only. Essentially DataURI can be used for the `src` attribute of corresponding html tags.
 
## Suggest
make the image/audio/video element support bytes argument and convert it to DataURI automatically.
this will be more effective to output small image/audio/video generated dynamically.

**Comment by rodja** [comment]
Thanks @liunux4odoo. This would be a great feature. Anybody interested in creating a pull request?

**Reply by falkoschindler** [reply]
Aren't Data URIs already supported for images? Or is it something different than base64-encoded images?
